### ISSUE TITLE
Rewrite some links in mirrored content?

### ISSUE THREAD

**Issue by FerociousTruth** [Discordius/Lesswrong2#180]
When LessWrong is mirroring content from another blog, perhaps links to other mirrored posts should be rewritten to stay within the site.  For example, on the post [Against Individual IQ Worries](https://www.lesserwrong.com/posts/AmaWMMWPzuQ62Ernf/against-individual-iq-worries) links to [Attitude vs. Altitude](http://slatestarcodex.com/2015/02/01/talents-part-2-attitude-vs-altitude/) on Scot's blog.  In this case I don't that that post is (yet?) mirrored to LessWrong.  But if it were, then I feel like maybe the link should change to point at the mirrored copy.  This fits with the goal of keeping LessWrong as the conversational locus.

**Comment by Discordius** [comment]
Yep, also agree with this, though this is definitely non-trivial since you need to parse all the links in a post and change them, ideally in a way that doesn't break if we don't end up having the original post.
